modules/deploy/partials/kubernetes/guides/create-storageclass.adoc

```diff
@@ -24,13 +25,22 @@ ifndef::env-eks[]
 helm repo add metal-stack https://helm.metal-stack.io
 helm repo update
 helm install csi-driver-lvm metal-stack/csi-driver-lvm \
+  --version 0.6.0 \
   --namespace csi-driver-lvm \
   --create-namespace \
   --set lvm.devicePattern='/dev/nvme[0-9]n[0-9]'
 ----
 endif::[]
 +
 The `lvm.devicePattern` property specifies the pattern that the CSI driver uses to identify available NVMe volumes on your worker nodes.
++
+[NOTE]
+====
+Version 0.6.0 of `csi-driver-lvm` is used to avoid volume-mounting issues with newer versions. Later versions enable some XFS filesystem features that are incompatible with the default kernels used in {k8s-managed-service}, leading to failed mounts. In recent `mkfs.xfs` updates, the `-i nrext64=1` option is enabled by default, which leads to the following error on older kernels:
```

Comment:
```suggestion
Version 0.6.0 is required to avoid volume-mounting issues caused by recent `mkfs.xfs` updates. Newer versions enable the `-i nrext64=1` option, triggering the following error on default {k8s-managed-service} kernels:
```